xgboost_ray/main.py

```diff
@@ -816,8 +852,14 @@ def train(params: Dict,
     checkpoint = _Checkpoint()  # Keep track of latest checkpoint
     current_results = {}  # Keep track of additional results
     actors = [None] * ray_params.num_actors  # All active actors
-    queue = Queue()  # Queue actor
-    stop_event = Event()  # Stop event actor
+
+    # Create the Queue and Event actors.
+    queue, stop_event = _create_communication_processes()
+
+    pg = _create_placement_group(cpus_per_actor, gpus_per_actor,
+                                 ray_params.resources_per_actor,
+                                 ray_params.num_actors)
+
```

Comment:
The main question I have here is if we should always use placement groups even if we're not using Tune. Does this make a difference in practice? If not using Tune we would probably want to spread actors as much as possible.

Reply:
Right, this is one of the assumptions made in the doc that if not using Tune, the cluster is fully utilized (all CPUs are being used by Xgboost Actors). In this case, it doesn't matter whether we use Pack, Spread, or no placement groups at all.

But you're right if we don't make that assumption, we would want to use Spread strategy. I added that in here since it was pretty straightforward.